src/ucp/core/ucp_worker.c

```diff
@@ -1046,7 +1046,7 @@ static void ucp_worker_close_ifaces(ucp_worker_h worker)
     UCS_ASYNC_BLOCK(&worker->async);
     for (iface_id = 0; iface_id < worker->num_ifaces; ++iface_id) {
         wiface = worker->ifaces[iface_id];
-        if (wiface->iface != NULL) {
+        if ((wiface != NULL) && (wiface->iface != NULL)) {
```

Comment:
the check for wiface->iface != NULL should be inside ucp_worker_iface_cleanup

Reply:
good point
done